ingestion/src/metadata/ingestion/ometa/ometa_api.py

```diff
@@ -415,13 +415,24 @@ def _get(
                 )
             return entity(**resp)
         except APIError as err:
-            logger.error(
-                "GET %s for %s." "Error %s - %s",
-                entity.__name__,
-                path,
-                err.status_code,
-                err,
-            )
+            if err.status_code == 404:
+                logger.info(
+                    "GET %s for %s." "Error %s - %s",
```

Comment:
Here instead of `Error`, we should say something like `Entity not found`

Reply:
Is it ok if I replace `Error` with `HTTP` so the error would be 

```
[2022-02-22 15:14:13,642] INFO     {metadata.ingestion.ometa.ometa_api:419} - GET DatabaseService for name/local_mysql. HTTP 404 - databaseService instance for local_mysql not found
```

or other way could be I just remove the `Error` word so the error would be.

```
[2022-02-22 15:14:13,642] INFO     {metadata.ingestion.ometa.ometa_api:419} - GET DatabaseService for name/local_mysql. databaseService instance for local_mysql not found
```

Reply:
Oh the first one looks great! +1 to `HTTP 404`, super informative